### ISSUE TITLE
Typo on dynamic-content/#moving-pointer page: aria-tooltip

### ISSUE THREAD

**Issue by carmacleod** [IBMa/equal-access#190]
The [dynamic-content/#moving-pointer](https://www.ibm.com/able/toolkit/develop/coding-interactions/dynamic-content/#moving-pointer) page says:
> Avoid using aria-tooltip which cannot contain interactive elements; instead use ARIA role=”dialog”.

Since `aria-tooltip` is not a thing, this sentence should read:
> Avoid using role=”tooltip” which cannot contain interactive elements; instead use ARIA role=”dialog”.

**Comment by tombrunet** [comment]
Included in https://github.ibm.com/IBMa/ibm.com-able-v2/pull/403

**Comment by tombrunet** [comment]
Deployed.
